logging/src/main/java/com/virtuslab/logger/PrefixedLambdaLogger.java

```diff
@@ -23,7 +23,7 @@
     className = classPath[classPath.length - 1];
   }
 
-  private String getLogMessagePrefix() {
+  private String getStackTraceAsString() {
```

Comment:
Whoops not this method should have been renamed?

Reply:
Whoops...